lib/rules/frame-title-unique.json

```diff
@@ -2,13 +2,14 @@
   "id": "frame-title-unique",
   "selector": "frame[title], iframe[title]",
   "matches": "frame-title-has-text-matches",
-  "tags": ["cat.text-alternatives", "best-practice"],
+  "tags": ["cat.text-alternatives", "wcag412", "wcag2a"],
   "actIds": ["4b1c6c"],
   "metadata": {
     "description": "Ensures <iframe> and <frame> elements contain a unique title attribute",
-    "help": "Frames should have a unique title attribute"
+    "help": "Frames must have a unique title attribute"
   },
   "all": [],
   "any": [],
-  "none": ["unique-frame-title"]
+  "none": ["unique-frame-title"],
+  "reviewOnFail": true
```

Comment:
Don't know why, but this isn't working, otherwise the integration tests would've failed. This is not something that can be changed without updating integration tests.